Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/signature/core/DscSignatureValidator.kt

```diff
@@ -8,10 +8,9 @@ import javax.inject.Inject
 @Reusable
 class DscSignatureValidator @Inject constructor() {
 
-    suspend fun isSignatureValid(dscData: DscData, certificateData: DccData<*>): Boolean {
+    suspend fun isSignatureValid(dscData: DscData, certificateData: DccData<*>) {
```

Comment:
As it does not return a Boolean any more, is... is misleading. Maybe just checkSignature(...)?

Reply:
Was changed to `validateSignature` by merging 2.7 into my branch.